uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java

```diff
@@ -430,8 +430,11 @@ private void queueRenameEvent(final WatchContext context,
                                                       final IndexerDispatcher dispatcher) throws DisposedException {
                             final Path sourcePath = context.getOldPath();
                             final Path destinationPath = context.getPath();
-                            dispatcher.offer(new RenamedFileEvent(sourcePath,
-                                                                  destinationPath));
+
```

Comment:
This method  `private void queueRenameEvent(final WatchContext context, final IndexerDispatcher dispatcher)` is not covered by tests.

Reply:
Hi, @domhanak thanks for your review. I have added some unit tests. I had to extract some code to be able to test it.